src/Diagram/stateManager/descriptionManager/index.ts

```diff
@@ -122,7 +122,10 @@ export class DescriptionManager {
         this.selectedNode &&
         descriptionObjectContent !== this.DESCRIPTION_PLACEHOLDER
       ) {
-        this.selectedNode.description = descriptionObjectContent;
+        this.selectedNode.description = descriptionObjectContent.replace(
```

Comment:
Either extract this de-HTML-ing logic to a named utility function or place a comment describing it.